ynr/apps/bulk_adding/static/bulk_adding/js/bulk.js

```diff
@@ -139,7 +139,7 @@ $(function(){
     }
 
     // Remove empty fieldsets
-    var always_show_extra = $("form#bulk_add_form")[0].dataset['winnerCount'] * 3;
+    var always_show_extra = $("form#bulk_add_form")[0].dataset.winnerCount * 3;
```

Comment:
I haven't had chance to check this out locally to test, but I dont think this would fix the 500 error I mentioned in an old comment that happens when a ballot does not have `winner_count` set? I think that will need a change in the view.

There was also another bug that I came across when testing originally, that from memory, was that if I filled in multiple candidates using the new "add a row" button, and then submitting the form, only the last candidate that I entered was being created/shown for review. But as mentioned it was a while ago I tested it locally so will need to go through again when I have chance!